README.md

```diff
@@ -145,10 +145,12 @@ Please [send a Pull Request](https://github.com/awslabs/aws-data-wrangler/edit/m
 - [Funcional Health Tech](https://www.funcionalcorp.com.br/) [[@webysther](https://github.com/webysther)]
 - [LINE TV](https://www.linetv.tw/) [[@bryanyang0528](https://github.com/bryanyang0528)]
 - [M4U](https://www.m4u.com.br/) [[@Thiago-Dantas](https://github.com/Thiago-Dantas)]
+- [nrd.io](https://nrd.io/) [[@mrtns](https://github.com/mrtns)]
```

Comment:
Is the url `https://nrd.io/` right? Seems to be offline for a while.

Reply:
It's right, there's just nothing there #stealthmode ;)